tests/test_end_to_end.py

```diff
@@ -19,7 +19,7 @@ def pipeline():
         yield p
 
 
-@pytest.mark.parametrize("target_chunks", [{"time": 1}, {"time": 2}, {"time": 3}])
+@pytest.mark.parametrize("target_chunks", [{"time": 1}, {"time": 2}, {"time": 3}, {'time':1, 'lon': 18}])
```

Comment:
Ensures that providing target chunks to additional dimension which are part of the data variable does not trigger the failure